<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CJLU内网有线登录 V1.1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_login">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_logout">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>注销</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>268</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_account">
         <property name="text">
          <string>账号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_account">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_account">
         <property name="text">
          <string>保存账号  </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_password">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_password">
         <property name="text">
          <string>保存密码  </string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>92</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;网络连接检测中。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Internet">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>92</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>访问互联网</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开发：14光电的浩宝宝   邮箱：625607188@qq.com</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_login</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_logout</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>logout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_password</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox_account</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>login()</slot>
  <slot>logout()</slot>
 </slots>
</ui>
